db/migrate/20230322195141_add_can_approve_careplan_permission.rb

```diff
@@ -0,0 +1,10 @@
+class AddCanApproveCareplanPermission < ActiveRecord::Migration[6.1]
+  def up
+    Role.ensure_permissions_exist
+    Role.reset_column_information
+  end
+
+  def down
+    remove_column :roles, :can_manage_accountable_care_organizations
```

Comment:
This doesn't match the name of the permission you added to role.rb, is that a mistake?